__tests__/read-file.test.ts

```diff
@@ -0,0 +1,54 @@
+import { readUserDataFile } from "@src/utils/read-file";
+import { User } from "@src/types";
+
+describe("Normal cases", () => {
+  {
+    test("[1]. Should return list of users when file content is correct", () => {
+      const expectData: User[] = [
+        {
+          name: "Nguyen",
+          phone: "0944324560",
+          address: "Thanh-Xuan",
+        },
+        {
+          name: "Toan",
+          phone: "0324324400",
+          address: "Nam-Tu-Liem",
+        },
+      ];
+      expect(
+        // Prepare users data file
+        readUserDataFile("./__tests__/test-data/test-data.txt")
+        // Expect list of Users
```

Comment:
không nên comment lưng chừng như này comment trước dòng code chính thôi em (dòng 19)
chỗ này chỉ cần comment đại loại kiểu: // Verify extracted user data from data file is correct